code/game/atoms_movable.dm

```diff
@@ -848,6 +848,9 @@
 /atom/movable/proc/do_attack_animation(atom/target, atom/tool)
 	set waitfor = 0
 
+	if(!tool)
```

Comment:
What they're saying is that there never should not be a tool, so they want an `ASSERT(tool)` instead so we know when the proc is used incorrectly.